sphinx/domains/cpp.py

```diff
@@ -3659,24 +3659,33 @@ def describe_signature(self, signode: TextElement, mode: str,
 
 
 class ASTTemplateParams(ASTBase):
-    def __init__(self, params: List[ASTTemplateParam]) -> None:
+    def __init__(self, params: List[ASTTemplateParam],
+                 requiresClause: Optional["ASTRequiresClause"] = None) -> None:
         assert params is not None
         self.params = params
+        self.requiresClause = requiresClause
 
-    def get_id(self, version: int) -> str:
+    def get_id(self, version: int, exclude_requires: bool = False) -> str:
```

Comment:
What is the reason for not including it in the ID?

Reply:
I added a comment to explain.